pages/profil.tsx

```diff
@@ -130,10 +138,36 @@ function Profil({ referentielAgences }: ProfilProps) {
     })
   }
 
+  function trackAccederAIMilo() {
+    trackPage({
+      structure: userStructureDimensionString(StructureConseiller.MILO),
+      customTitle: 'Accès i-milo',
+    })
+  }
+
   useMatomo(trackingLabel, aDesBeneficiaires)
 
   return (
     <>
+      {showInformationEmailManquant && (
+        <div className='mb-6'>
+          <InformationMessage
+            label='Votre adresse mail n’est pas renseignée'
```

Comment:
```suggestion
            label='Votre adresse e-mail n’est pas renseignée'
```
dans l'administration française cf. https://zeroheight.com/2f9442f46/p/473c3c-conseiller/b/75adeb